Scripts/Skills/DetectHidden.cs

```diff
@@ -80,6 +81,20 @@ protected override void OnTarget(Mobile src, object targ)
                     }
 
                     inRange.Free();
+                    
+                    IPooledEnumerable ExodusChestInRange = src.Map.GetItemsInRange(p, 3);
```

Comment:
You can take advantage of LINQ and foreach implicit disposal to clean up this code:

```
foreach (var chest in src.Map.GetItemsInRange(p, 3).OfType<ExodusChest>().Where(it => !it.Visible))
{
	chest.Visible = true;
	chest.StartDeleteTimer();
}
```